2021-04-17 11:11:17,156 INFO    MainThread:35030 [wandb_setup.py:_flush():69] setting env: {}
2021-04-17 11:11:17,156 INFO    MainThread:35030 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-17 11:11:17,156 INFO    MainThread:35030 [wandb_init.py:_log_setup():336] Logging user logs to /home/ubuntuadm/trainseg/utils/wandb/run-20210417_111117-19cbnosb/logs/debug.log
2021-04-17 11:11:17,156 INFO    MainThread:35030 [wandb_init.py:_log_setup():337] Logging internal logs to /home/ubuntuadm/trainseg/utils/wandb/run-20210417_111117-19cbnosb/logs/debug-internal.log
2021-04-17 11:11:17,156 INFO    MainThread:35030 [wandb_init.py:init():369] calling init triggers
2021-04-17 11:11:17,156 INFO    MainThread:35030 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-17 11:11:17,156 INFO    MainThread:35030 [wandb_init.py:init():418] starting backend
2021-04-17 11:11:17,156 INFO    MainThread:35030 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-17 11:11:17,174 INFO    MainThread:35030 [backend.py:ensure_launched():123] starting backend process...
2021-04-17 11:11:17,181 INFO    MainThread:35030 [backend.py:ensure_launched():127] started backend process with pid: 35045
2021-04-17 11:11:17,189 INFO    MainThread:35030 [wandb_init.py:init():423] backend started and connected
2021-04-17 11:11:17,190 INFO    MainThread:35030 [wandb_init.py:init():465] updated telemetry
2021-04-17 11:11:17,196 INFO    MainThread:35030 [wandb_init.py:init():484] communicating current version
2021-04-17 11:11:18,029 INFO    MainThread:35030 [wandb_init.py:init():489] got version response 
2021-04-17 11:11:18,029 INFO    MainThread:35030 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-17 11:11:18,237 INFO    MainThread:35030 [wandb_init.py:init():522] starting run threads in backend
2021-04-17 11:11:19,558 INFO    MainThread:35030 [wandb_run.py:_console_start():1524] atexit reg
2021-04-17 11:11:19,558 INFO    MainThread:35030 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-17 11:11:19,559 INFO    MainThread:35030 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-17 11:11:19,584 INFO    MainThread:35030 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-17 11:11:19,585 INFO    MainThread:35030 [wandb_init.py:init():546] run started, returning control to user process
2021-04-17 11:11:24,499 INFO    MainThread:35030 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-17 11:11:24,500 INFO    MainThread:35030 [wandb_run.py:_restore():1466] restore
2021-04-17 11:11:26,704 INFO    MainThread:35030 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 834
  total_bytes: 834
}

2021-04-17 11:11:27,293 INFO    MainThread:35030 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 834
  total_bytes: 3395
}

2021-04-17 11:11:27,646 INFO    MainThread:35030 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3395
  total_bytes: 3395
}

2021-04-17 11:11:27,748 INFO    MainThread:35030 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3395
  total_bytes: 3395
}

2021-04-17 11:11:27,850 INFO    MainThread:35030 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3395
  total_bytes: 3395
}

2021-04-17 11:11:27,952 INFO    MainThread:35030 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3395
  total_bytes: 3395
}

2021-04-17 11:11:28,054 INFO    MainThread:35030 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3395
  total_bytes: 3395
}

2021-04-17 11:11:29,373 INFO    MainThread:35030 [wandb_run.py:_show_files():1838] logging synced files
